Sockets/BackEndMonitoringSockets/include/CAcceptorWrapper.h

```diff
@@ -10,18 +10,18 @@ class CAcceptorWrapper
 {
 public:
 	CAcceptorWrapper(int port, const std::string& ip_address, 
-		size_t num_threads, CEvent& event, std::shared_ptr<CLogger> logger);
+		CEvent& event, std::shared_ptr<CThreadPool> m_pool,
+		std::shared_ptr<CLogger> logger);
 	void StartServer();
-
+	bool StopSocket();
 private:
-	std::unique_ptr<CThreadPool> InitThreadPool(int num_threads, CEvent& event);
 	std::unique_ptr<CAcceptor> InitAcceptor(int port, 
 		const std::string& address);
 	std::unique_ptr<CServiceConnectionHandler> InitServiceHandler(int handle);
 	std::unique_ptr<CSocketWrapper> InitSocketWrapper(int handle);
 
 	CEvent& m_event;
-	std::unique_ptr<CThreadPool> m_pool;
+	std::shared_ptr<CThreadPool> m_pool;
 	std::unique_ptr<CAcceptor> m_server_acceptor;
 	std::unique_ptr<CServiceConnectionHandler> m_service_handler;
 	std::unique_ptr<CSocketWrapper> m_stream;
```

Comment:
if pointer - add prefix p_ (example: m_p_pool)

Reply:
@Serwlashev, this too.